doc/source/overview.rst

```diff
@@ -0,0 +1,81 @@
+.. _overview:
+
+Overview
+========
+
+.. note:: **Abstract**
+
+   This document provides a conceptual overview about the steps of creating
+   an image description with keg which can be used to build an appliance
+   with the `KIWI <https://osinside.github.io/kiwi/>`__ appliance builder.
+
+Conceptual Overview
+-------------------
+
+Keg is a tool which helps to create and manage image descriptions suitable
+for the `KIWI <https://osinside.github.io/kiwi/>`__ appliance builder. Its
+main use case is to keep control over a larger amount of image descriptions
```

Comment:
s/larger/large ?

larger than what ? does it mean `larger amount of image descriptions than KIWI ` ?

Reply:
See suggestion for keg.rst where this construct shows up first.

Reply:
done